### ISSUE TITLE
Public posts list

### ISSUE THREAD

**Issue by zerosonesfun** [PrivateBin/PrivateBin#555]
I have a suggestion. I feel like this could be turned into a basic forum of sorts. All that would need to happen is, when someone creates a public post with discussion enabled, on the main page or some page, there is a list of the latest public posts. This way, people can jump in and join various discussions without having the url.

It could be an extra settings check box. As you are posting, if you choose public and discussion, a final option appears that you can check called “display on homepage,” or something like that.

You’d still have all of the private features, plus a part of the site that features public posts which can be explored.

**Comment by elrido** [comment]
I have seen some instances doing this by redirecting the requests for the web root (path / on that domain) to a paste with discussions enabled, explaining the "forum" rules and everyone posting their "threads" (links to other pastes) underneath in the comments. This is in part what #2, #5 and #255 would help foster.

At it's core, PrivateBin intends to protect the service provider from being made responsible for content hosted on their instance. If they can't decrypt the pastes, they have plausible deniability on the contents until that moment that they receive a takedown request. For a public paste, the key of these pastes would have to be stored on the server, readable to anyone, including the service provider. I.e. in France this would then force them to have to check the contents or they become responsible for them.

Long story short: This would should be an optional feature and it can already be manually implemented by offering an "index" paste, i.e. via the `notice` configuration setting or editing the template and inserting that link.

With #2 one could split the responsibility: Use one instance for the "index" paste that links to other instance(s). Neither service provider could be made responsible for having to investigate content that is on another instance.

**Comment by rugk** [comment]
I agree this can already be made possible with a notice and potentially a paste; also useful to combined with #2 or so.

As such, I'd tend to close this issue as a wontfix, as I do not feel fine implementing some public posts list in a service for private posts.
I mean, it would require to collect decryption links and this is something we really don't want to do.

As such, if you want a _public_ service, don't use _Private_ Bin, I guess there are better alternatives. :smile: 
Or just make a paste that contains all the links you want, maybe also as comments.

Anyway, I consider an explicit feature for this off-topic for this project.
